Mage/src/main/java/mage/abilities/keyword/EvokeAbility.java

```diff
@@ -25,7 +25,7 @@ public EvokeAbility(String manaString) {
     public EvokeAbility(Cost cost) {
         super(EVOKE_KEYWORD, REMINDER_TEXT, cost);
         Ability ability = new ConditionalInterveningIfTriggeredAbility(
-                new EntersBattlefieldTriggeredAbility(new SacrificeSourceEffect()),
+                new EntersBattlefieldTriggeredAbility(new SacrificeSourceEffect(true)),
                 EvokedCondition.instance, "Sacrifice {this} when it enters the battlefield and was evoked.");
```

Comment:
update text to "When this permanent enters the battlefield, if its evoke cost was paid, its controller sacrifices it." per rules reference 702.74a